Random-Name-Picker.iml

```diff
@@ -12,6 +12,7 @@
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <excludeFolder url="file://$MODULE_DIR$/.gradle" />
+      <excludeFolder url="file://$MODULE_DIR$/build" />
```

Comment:
Hmmm, I need to remove this from the repository. It's an IDE specific file.

Reply:
Oh, I keep forgetting it, even in the new commit I forgot it, sorry